package.json

```diff
@@ -1,79 +1,82 @@
 {
-  "name": "@frctl/fractal",
-  "version": "1.2.1",
-  "description": "A tool to help you build and document web component libraries and then integrate them into your projects.",
-  "main": "src/fractal.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/frctl/fractal.git"
-  },
-  "author": "Mark Perkins <mark@clearleft.com> (http://clearleft.com/)",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/frctl/fractal/issues"
-  },
-  "homepage": "https://github.com/frctl/fractal",
-  "dependencies": {
-    "@allmarkedup/fang": "^1.0.0",
-    "@frctl/handlebars": "^1.1.3",
-    "@frctl/mandelbrot": "^1.1.0",
-    "anymatch": "^2.0.0",
-    "bluebird": "^3.5.2",
-    "browser-sync": "^2.26.4",
-    "chalk": "^1.1.3",
-    "chokidar": "^3.3.1",
-    "cli-table3": "^0.5.0",
-    "co": "^4.6.0",
-    "columnify": "^1.5.4",
-    "express": "^4.14.0",
-    "fs-extra": "^0.30.0",
-    "globby": "^6.0.0",
-    "gray-matter": "^2.0.2",
-    "handlebars": "^4.0.13",
-    "highlight.js": "^9.5.0",
-    "inquirer": "^1.1.2",
-    "istextorbinary": "^2.1.0",
-    "js-yaml": "^3.13.1",
-    "liftoff": "^2.3.0",
-    "lodash": "^4.17.11",
-    "log-update": "^1.0.2",
-    "marked": "^0.7.0",
-    "mime": "^1.3.4",
-    "minimist": "^1.2.2",
-    "mixwith": "^0.1.1",
-    "nunjucks": "^3.2.1",
-    "path-to-regexp": "^1.5.3",
-    "portscanner": "^1.0.0",
-    "readable-stream": "^2.1.4",
-    "require-all": "^2.0.0",
-    "semver": "^5.3.0",
-    "throat": "^3.0.0",
-    "update-notifier": "^2.0.0",
-    "vinyl": "^1.2.0",
-    "vorpal": "^1.12.0"
-  },
-  "bin": {
-    "fractal": "./bin/fractal"
-  },
-  "engines": {
-    "node": ">= 4.4.7"
-  },
-  "scripts": {
-    "test": "mocha --require test/support/env --reporter spec",
-    "coverage": "nyc --reporter=lcov --reporter=text  npm run test"
-  },
-  "devDependencies": {
-    "chai": "^4.1.1",
-    "chai-as-promised": "^7.1.1",
-    "istanbul": "^0.4.4",
-    "mocha": "^5.0.3",
-    "mock-fs": "^4.6.0",
-    "nyc": "^15.0.1",
-    "pre-commit": "^1.1.3",
-    "proxyquire": "^1.7.10",
-    "sinon": "^9.0.1"
-  },
-  "publishConfig": {
-    "access": "public"
-  }
+    "name": "@frctl/fractal",
```

Comment:
not sure about this indentation change... i know in the past there have been some issues with npm forcing 2-space indentation in package.json, however these seem to be solved now. 2-space should still be pretty standard for this specific file, i think.

Our .editorconfig also suggests using 2-space for json files - if we're going to change the indentation guidelines, we should change it there as well.

Reply:
Whoops, this was just VSCode trying to be clever. I switched it back.